fiftyone/core/odm/dataset.py

```diff
@@ -22,12 +22,19 @@
     TargetsField,
 )
 
-from .document import Document, EmbeddedDocument, BaseEmbeddedDocument
+from .document import Document
+from .embedded_document import EmbeddedDocument, BaseEmbeddedDocument
 from .runs import RunDocument
 
 
 def create_field(
```

Comment:
`create_field()` is now a recursive method

Reply:
A notable limitation of current schemas is a `ListField(ListField(OtherField))` cannot be described by the data model, i.e. a `SampleFieldDocument`. This was already the case, of course.